apps-rendering/README.md

```diff
@@ -35,7 +36,19 @@ The Apps Rendering development server supports the following routes for testing
 - `/(uk|us|au|international)/rendered-items/path/to/content`
   - e.g [http://localhost:8080/au/rendered-items/food/2020/mar/15/easter-taste-test-dan-lepard-hot-cross-bun-milk-dark-chocolate-mini-eggs-bunny-sloth](http://localhost:8080/au/rendered-items/food/2020/mar/15/easter-taste-test-dan-lepard-hot-cross-bun-milk-dark-chocolate-mini-eggs-bunny-sloth)
 
-Additionally, each route above can take a `?editions` query parameter to render the article as for the Editions app.
+Additionally, each route above can take a `?editions` query parameter to render
+the article as for the Editions app.
+
+We also recommend testing articles in the mobile device simulators.
+You can start the development server on port `3030` (the port number expected by
+the device simulator) with `yarn watch:simulator`. The development server
+also supports a specific route for
+testing with the device simulator:
+
+- `/AppsArticle?url=https://www.theguardian.com/food/2020/mar/15/easter-taste-test-dan-lepard-hot-cross-bun-milk-dark-chocolate-mini-eggs-bunny-sloth`
+  - e.g [http://localhost:3030/AppsArticle?url=https://www.theguardian.com/food/2020/mar/15/easter-taste-test-dan-lepard-hot-cross-bun-milk-dark-chocolate-mini-eggs-bunny-sloth](http://localhost:3030/AppsArticle?url=https://www.theguardian.com/food/2020/mar/15/easter-taste-test-dan-lepard-hot-cross-bun-milk-dark-chocolate-mini-eggs-bunny-sloth)
```

Comment:
Is this still the correct URL? Am I right in thinking you've updated the code to use the path rather than the query param?

Reply:
oops yes, I need to update this as well!